.github/workflows/test-suite.yml

```diff
@@ -84,4 +84,42 @@ jobs:
     if: ${{ !cancelled() }}
     secrets: inherit
     uses: ./.github/workflows/cleanup-test-env.yml
-  
\ No newline at end of file
+  
+  slack-notification:
+    needs: [mig-tests, acc-tests, clean-after]
+    if: ${{ (github.event_name == 'schedule' || github.event_name == 'workflow_dispatch') && (needs.mig-tests.result == 'failure' || needs.acc-tests.result == 'failure')}}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Send Slack message
+        id: slack
+        uses: slackapi/slack-github-action@v1.25.0
+        with:
+          payload: |
```

Comment:
Hardcoding message here because this notification will only be triggered when test fail, opposed to Compatibility matrix which will be sent both when it succeeds and fails